source/common/runtime/runtime_features.cc

```diff
@@ -33,6 +33,7 @@ RUNTIME_GUARD(envoy_reloadable_features_enable_grpc_async_client_cache);
 RUNTIME_GUARD(envoy_reloadable_features_fix_added_trailers);
 RUNTIME_GUARD(envoy_reloadable_features_handle_stream_reset_during_hcm_encoding);
 RUNTIME_GUARD(envoy_reloadable_features_http2_allow_capacity_increase_by_settings);
+RUNTIME_GUARD(envoy_reloadable_features_http2_new_codec_wrapper);
```

Comment:
This should go back in the release notes, right? Do you mind adding one? Thank you.

/wait

Reply:
Sorry for the oversight. I filled in the "Release notes" field above. Is there any other file or field I need to edit?

Reply:
Envoy release notes are in docs/root/version_history/current.rst
feel free to ping me internally for what to add and where if it's not obvious. :-)

Reply:
Done, thanks.